src/components/SideMenu/sidemenu.import.less

```diff
@@ -11,74 +10,73 @@
       cursor: pointer;
     }
   }
+  .sidemenu__background {
+    display: none;
+    background-color: #000;
+    opacity: 0.4;
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    z-index: 3;
+  }
   .sidemenu__nav {
     height: 100%;
-    width: 0;
+    width: 250px;
     position: fixed;
-    z-index: 100;
+    z-index: 4;
     top: 0;
-    right: 0;
-    background-color: #333;
-    overflow: hidden;
+    right: -250px;
+    background-color: #180744; // overflow: hidden;
```

Comment:
@nicholasmiller1 Remember to use color variable names here, defined in variables.less. (I think those are set up now)